kitsune/wiki/templates/wiki/edit.html

```diff
@@ -22,6 +22,12 @@
 
     <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
       <h1>{{ _('<em>Editing</em> {title}')|fe(title=document.title) }}</h1>
+      {% if document.needs_change %}
+        <p>
+          {% trans comment=document.needs_change_comment %} <b>Needs Change:</b> {{comment}} {% endtrans %}
```

Comment:
`<b/>` isn't a good tag to use because it implies a certain style and that should be defined in CSS. `<strong/>` is better.

But I think it would be best to be consistent with the form below. Something like:

```
<label>{{ _('Needs Change:') }}</label>
<p>{{ comment }}</p>
```

Then make sure that the label is bolded via CSS.